common-items.lic

```diff
@@ -119,9 +119,44 @@ module DRCI
     result =~ /inside/
   end
 
+  # Is the item in either hand?
   def in_hands?(item)
-    item_regex = item.split.size > 1 ? /.*\b#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
-    item_regex =~ "#{DRC.right_hand}#{DRC.left_hand}"
+    in_hand?(item, 'either')
+  end
+
+  # Is the item in your left hand?
+  def in_left_hand?(item)
+    in_hand?(item, 'left')
+  end
+
+  # Is the item in your right hand?
+  def in_right_hand?(item)
+    in_hand?(item, 'right')
+  end
+
+  # Checks if the item is in one or more hands.
+  # Hand options are: left, right, either, both.
+  def in_hand?(item, whichHand = 'either')
+    item_regex = item.split.size > 1 ? /#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
+    item_found = false
+    case whichHand.downcase
+    when 'left'
+      # Your holding the item in your left hand?
+      item_found = ("#{DRC.left_hand}" =~ item_regex) != nil
+    when 'right'
+      # Your holding the item in your right hand?
+      item_found = ("#{DRC.right_hand}" =~ item_regex) != nil
+    when 'either'
+      # At least one hand is holding the item?
+      item_found = in_hand?(item, 'left') || in_hand?(item, 'right')
+    when 'both'
+      # You're holding two of the same item, one in each hand?
+      item_found = in_hand?(item, 'left') && in_hand?(item, 'right')
+    else
+      DRC.message("Unknown hand: #{whichHand}. Valid options are: left, right, either, both")
+      item_found = false
+    end
+    return item_found
```

Comment:
You can get rid of item_found entirely and returns, ruby methods return the value of the last evaluated statement by default. e.g.

```
irb(main):001:0> def test(item)
irb(main):002:1>   case item
irb(main):003:2>   when 'bob'
irb(main):004:2>     true
irb(main):005:2>   when 'tim'
irb(main):006:2>     'tod'
irb(main):007:2>   else
irb(main):008:2>     false
irb(main):009:2>   end
irb(main):010:1> end
=> :test
irb(main):011:0> test('bob')
=> true
irb(main):012:0> test('tim')
=> "tod"
irb(main):013:0> test('joe')
=> false
```